src/RacingGame.js

```diff
@@ -0,0 +1,70 @@
+import { inputHandler, outputHandler } from './views/index.js';
+import { GAME_MESSAGES, RULES } from './constants/index.js';
+import {
+  carNameParser,
+  randomNumber,
+  validateCarNames,
+} from './utils/index.js';
+import { Car } from './models/index.js';
+
+class RacingGame {
+  #cars;
+  #tryCount;
+
+  async startGame() {
+    await this.#initialize();
+    await this.#play();
+    await this.#finish();
+  }
+
+  async #initialize() {
+    const carNames = await inputHandler.carNameInput();
+    const parsedNames = carNameParser(carNames);
+    validateCarNames(parsedNames);
+
+    this.#cars = parsedNames.map((name) => new Car(name));
+    this.#tryCount = Number(await inputHandler.racingTryCountInput());
+  }
+
+  async #play() {
+    outputHandler.printMessage(GAME_MESSAGES.RESULT);
+
+    for (let i = 0; i < this.#tryCount; i++) {
+      this.#playRound();
+      outputHandler.printRoundResult(this.#cars);
+      outputHandler.printMessage('');
+    }
+  }
+
+  async #finish() {
+    const winners = this.#getWinners();
+    outputHandler.printMessage(`${GAME_MESSAGES.WINNER}${winners.join(', ')}`);
```

Comment:
저는 이런 경우에는 이렇게 사용하는 것 같은데 소은님은 템플릿 리터럴 문법을 사용하는 기준이 있으신가요??
```suggestion
    outputHandler.printMessage(GAME_MESSAGES.WINNER + winners.join(', '));
```

저는 개인적으로 템플릿 리터럴 문법을 사용할 때 중간에 어떤 문자열이 추가되는 경우에 사용하는 것 같아요!
```js
const a = "안녕";
const b = "홍길동";
console.log(`${a} ${b}`); // 중간에 공백이 있어서 템플릿 리터럴 문법 사용
console.log(`${a}-${b}`); // 중간에 다른 문자열이 있어서 템플릿 리터럴 문법 사용
console.log(a + b); // + 연산자로만 해결할 수 있다면 + 연산자 사용
```